packages/cli/src/generator.ts

```diff
@@ -234,93 +234,102 @@ export function getNamedImportNodes(
+      );
 
-  const statements: Statement[] = [];
+      statements.push(namedImport);
+    }
 
-  if (importNames.detected.length > 0) {
-    // Create a new named import node for the detected imports.
-    const namedImport = factory.createImportDeclaration(
+    // Create a new default import node.
+    const defaultImport = factory.createImportDeclaration(
       node.modifiers,
       factory.createImportClause(
         false,
+        factory.createIdentifier(importIdentifier),
         undefined,
-        factory.createNamedImports(
-          importNames.detected.map(({ propertyName, name }) =>
-            factory.createImportSpecifier(
-              false,
-              propertyName ? factory.createIdentifier(propertyName) : undefined,
-              factory.createIdentifier(name),
-            ),
-          ),
-        ),
       ),
       node.moduleSpecifier,
     );
 
-    statements.push(namedImport);
-  }
-
-  // Create a new default import node.
-  const defaultImport = factory.createImportDeclaration(
-    node.modifiers,
-    factory.createImportClause(
-      false,
-      factory.createIdentifier(importIdentifier),
+    // Create a variable declaration for the undetected import names.
+    const variableStatement = factory.createVariableStatement(
       undefined,
-    ),
-    node.moduleSpecifier,
-  );
-
-  // Create a variable declaration for the undetected import names.
-  const variableStatement = factory.createVariableStatement(
-    undefined,
-    factory.createVariableDeclarationList(
-      [
-        factory.createVariableDeclaration(
-          factory.createObjectBindingPattern([
-            ...importNames.undetected.map(({ propertyName, name }) =>
-              factory.createBindingElement(undefined, propertyName, name),
-            ),
-          ]),
-          undefined,
-          undefined,
-          factory.createIdentifier(importIdentifier),
-        ),
-      ],
-      // eslint-disable-next-line no-bitwise
-      NodeFlags.Const,
-    ),
-  );
+      factory.createVariableDeclarationList(
+        [
+          factory.createVariableDeclaration(
+            factory.createObjectBindingPattern([
+              ...importNames.undetected.map(({ propertyName, name }) =>
+                factory.createBindingElement(undefined, propertyName, name),
+              ),
+            ]),
+            undefined,
+            undefined,
+            factory.createIdentifier(importIdentifier),
+          ),
+        ],
+        // eslint-disable-next-line no-bitwise
+        NodeFlags.Const,
+      ),
+    );
 
-  statements.push(defaultImport, variableStatement);
+    statements.push(defaultImport, variableStatement);
 
-  return statements;
+    return statements;
+  } catch {
+    // If there is an error, return the node as is. This may occur if the module
+    // cannot be parsed by `cjs-module-lexer`, e.g., in the case of React
+    // Native, which resolves to a Flow file.
+    return node;
```

Comment:
I wonder if this will cause other problems and require some sort of config down the line

Reply:
Time will tell 😅